pkg/cert/v1beta1/cert.go

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Let's keep the name `pkg/cert-generator` to be consistent.

Reply:
It makes sense.
I fixed it.